src/ofdm.c

```diff
@@ -209,7 +209,7 @@ struct OFDM *ofdm_create(const struct OFDM_CONFIG *config) {
         ofdm->codename = "HRA_112_112";
         ofdm->amp_est_mode = 0;
         ofdm->tx_bpf_en = true;
-        ofdm->amp_scale = 217E3;
+        ofdm->amp_scale = 245E3;
```

Comment:
This needs to stay the same as master, as it's been carefully set up to minimise PAPR when 700D is used with compression.

The tests still pass, as both stm32 and x86 will use the same `amp_scale` :+1: